functions/src/Services/User.ts

```diff
@@ -1,10 +1,39 @@
 import { User, userConverter } from '../Models/User';
-import { db } from '../index';
+import { firestore } from 'firebase-admin';
+
+/**
+ * Handles retrieving the user collection with the converter.
+ */
+function getUserCollection() {
+	return firestore().collection('users').withConverter(userConverter);
+}
 
 export const createUser = async (user: User) => {
-	const userCollection = db.collection('users').withConverter(userConverter);
+	const userCollection = getUserCollection().withConverter(userConverter);
```

Comment:
why are you call withConverter() twice you already call it in the getUserCollection() method